rlbot_gui/bot_management/bot_creation.py

```diff
@@ -15,37 +17,38 @@ def convert_to_filename(text):
     """
     import unicodedata
     normalized = unicodedata.normalize('NFKD', text).encode('ascii', 'ignore').decode()
-    valid_chars = "-_.() %s%s" % (string.ascii_letters, string.digits)
+    valid_chars = '-_.() %s%s' % (string.ascii_letters, string.digits)
```

Comment:
use str.format here instead since we're using python3.

Reply:
Used f-strings as we are already at python 3.6 :)